packages/e2e-tests/test/repl-helpers.ts

```diff
@@ -89,6 +89,7 @@ const fakeExternalEditor = async ({
       if (${JSON.stringify(flags ?? '')}) {
         assert.deepStrictEqual((${JSON.stringify(
           flags
+          // eslint-disable-next-line no-useless-escape
```

Comment:
I this case I think the no-useless-escape rule was a false positive. It doesn't seem to know about `\s`.

Reply:
It’s because this whole section is inside a template string, so the actual regex ends up being `/s+/`, which is of course actually wrong.

This might be another good use case for `String.raw`? Either way, not a false positive

Reply:
Aah I see now. OMG that code is hard to read.